.github/workflows/continuous.yml

```diff
@@ -163,29 +163,28 @@ jobs:
         threading: [TBB]
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3.5.3
         with:
           fetch-depth: 10
 
       - name: Install Ninja
         uses: seanmiddleditch/gha-setup-ninja@master
 
       - name: Stetup Conda
-        uses: s-weigand/setup-conda@v1
+        uses: s-weigand/setup-conda@v1.2.1
         with:
           update-conda: true
           conda-channels: anaconda, conda-forge
 
       - name: Install Dependencies
         run: |
-          conda install libblas libcblas suitesparse -y
```

Comment:
why?

Reply:
We build SuiteSparse from source we do not need to install it.